### PR TITLE
add new changes to README to test rebase and merge

### PR DESCRIPTION
**Pull request by Yingying0618**
Test to merge branchtwo to master, with rebase

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Test
 this is a test
 test conflict
+
+#add a new commit to master branch to test rebase and merge
```
Comment (fastlotin2hp) [inline README.md:5]:
```suggestion
#add a new commit to master branch to test rebase and merge
```
Does it work? Eventually? What have you changed? 
Would share it with detail steps in this note?Thanks!

Comment (Yingying0618) [reply README.md:5]: OKay!

**Comment by Yingying0618** [comment]
Add a rebase and merge commit

**Comment by Yingying0618** [comment]
Really don't have merged information on the master branch, so rebase doesn't keep merge info.

**Pull request merged**